src/aind_behavior_dynamic_foraging/DataSchemas/task_logic.py

```diff
@@ -1,20 +1,43 @@
 from __future__ import annotations
 
 from enum import Enum
-from typing import Literal
+from typing import Annotated, List, Literal, Union, Optional
+from pydantic import BaseModel, Field, RootModel
 
 from aind_behavior_services.task_logic import AindBehaviorTaskLogicModel, TaskParameters
-from pydantic import Field
+from aind_behavior_dynamic_foraging.DataSchemas.fiber_photometry import Fiber_Photometry
+from aind_behavior_dynamic_foraging.DataSchemas.optogenetics import Optogenetics
+from aind_behavior_dynamic_foraging.DataSchemas.ephys import Ephys
 
-__version__ = "0.0.0"
+__version__ = "0.1.0"
 
 
-class AdvancedBlockMode(str, Enum):
-    """ Modes for advanced block """
+class NeuralExperimentTypes(RootModel):
```

Comment:
I would probably call this "PhysiologyExperimentTypes" as future experiments may not be neural